hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -61,15 +61,20 @@
 # reCaptcha settings
 RECAPTCHA_DOMAIN = "www.recaptcha.net"
 
+RECAPTCHA_TEST_PUBLIC_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+RECAPTCHA_TEST_PRIVATE_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+
 if DEBUG:
     # These are special keys which will always allow requests to pass
     # verification
-    RECAPTCHA_PUBLIC_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
-    RECAPTCHA_PRIVATE_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+    RECAPTCHA_PUBLIC_KEY = RECAPTCHA_TEST_PUBLIC_KEY
+    RECAPTCHA_PRIVATE_KEY = RECAPTCHA_TEST_PRIVATE_KEY
     SILENCED_SYSTEM_CHECKS = ["captcha.recaptcha_test_key_error"]
 else:
-    RECAPTCHA_PUBLIC_KEY = os.environ["RECAPTCHA_PUBLIC_KEY"]
-    RECAPTCHA_PRIVATE_KEY = os.environ["RECAPTCHA_PRIVATE_KEY"]
+    # If the default test keys are used, the captcha package will create a system
+    # check warning.
```

Comment:
@grahamhoyes where is this warning stored?

Reply:
> @grahamhoyes where is this warning stored?

It's part of the django-recaptcha package: https://github.com/praekelt/django-recaptcha/blob/develop/captcha/checks.py.

In debug mode, we silence it with the `SILENCED_SYSTEM_CHECKS` line above

Reply:
I see